shapash/utils/stability.py

```diff
@@ -0,0 +1,287 @@
+import numpy as np
+import pandas as pd
+import plotly.colors
+import re
+from sklearn.preprocessing import normalize
+
+
+def _df_to_array(instances):
+    """Transform inputs into arrays
+
+    Parameters
+    ----------
+    instances : DataFrame, Series or array
+        Input data
+
+    Returns
+    -------
+    instances : array
+        Transformed features
+    """
+    if isinstance(instances, pd.DataFrame):
+        return instances.values
+    elif isinstance(instances, pd.Series):
+        return np.array([instances.values])
+    else:
+        return instances
+
+
+def _compute_distance(x1, x2, mean_vector, epsilon=0.0000001):
+    """Compute distances between data points by using L1 on normalized data : sum(abs(x1-x2)/(mean_vector+epsilon))
+
+    Parameters
+    ----------
+    x1 : array
+        First vector
+    x2 : array
+        Second vector
+    mean_vector : array
+        Each value of this vector is the std.dev for each feature in dataset
+
+    Returns
+    -------
+    diff : float
+        Returns :math:`\\sum(\\frac{|x1-x2|}{mean\_vector+epsilon})`
+    """
+    diff = np.sum(np.abs(x1 - x2) / (mean_vector + epsilon))
```

Comment:
Warning: Division by 0

Reply:
Should I just remove the warning as this division by 0 is handled anyway ?

Reply:
can we use np.divide as suggested by this post stackoverflow : https://stackoverflow.com/questions/29950557/ignore-divide-by-0-warning-in-numpy